examples/squeezenet/SqueezeNet.js

```diff
@@ -407,8 +717,27 @@ class SqueezeNet {
           console.warn(`    ${node.opType} is not supported.}`);
         }
       }
-      if (opCode)
+      if (typeof opCode !== 'undefined') {
         this._model.addOperation(opCode, inputs, outputs);
+      }
+      if (i === graph.node.length - 1 && node.opType !== 'Softmax') { // last node in graph?
```

Comment:
Some object detection models might not have `softmax` at the last node. Do you have better way to add it? Perhaps create a configuration for specific model post-processing?

Reply:
yes, i proposed a way to handle this.
https://github.com/intel/webml-polyfill/pull/266#discussion_r228756112

Reply:
I have added an option to the constructor in https://github.com/intel/webml-polyfill/pull/266/commits/c8cc37c48d8441e62eaad7f43e3c0fb75f6810cd

If `modelOptions.softmax` is set to true, the importer will append a softmax layer called `softmax_appended` when the last layer is not a softmax. However, if this option is set to true but the model has already got a softmax layer, it won't do anything.

Reply:
looks good.